posts/test_post_uploading.py

```diff
@@ -69,12 +74,18 @@ def assert_posts_are_equal(self, first, second):
         self.assertEqual(first["files"], second["files"])
 
     def setUp(self):
-        """Set up mock DB for testing"""
+        """Set up mocks for testing."""
+        # mock db
         self.mock_collection = mongomock.MongoClient().db.collection
+        # mock Google storage module for uploading to storage bucket
+        app.storage = mock.MagicMock(spec=app.storage)
+        app.storage.Client().get_bucket().blob().public_url = MOCK_FILE_URL
 
     def test_full_upload(self):
         """Can upload a full post object with both text and files."""
         app.upload_new_post_to_db(VALID_POST_FULL, self.mock_collection)
+        (app.storage.Client().get_bucket().blob().upload_from_file
+         .assert_called())
```

Comment:
You might want to cache the result from `app.storage.Client()` in `self` (somewhere in `setUp`), which would probably make this fit on one line.

Reply:
Done https://github.com/knative-portability/large-events/pull/20/commits/5615d00b24da42270add718a294db18cfb8d9170